pkg/terra/config.go

```diff
@@ -18,7 +18,12 @@ var defaultConfigSet = configSet{
 	BlocksUntilTxTimeout:  10,
 	ConfirmPollPeriod:     time.Second,
 	FallbackGasPriceULuna: sdk.MustNewDecFromStr("0.015"),
-	GasLimitMultiplier:    1.5,
+	// This is high since we simulate before signing the transaction.
+	// There's a chicken and egg problem: need to sign to simulate accurately
+	// but you need to specify a gas limit when signing.
+	// TODO: Determine how much gas a signature adds and then
+	// add that directly so we can be more accurate.
+	GasLimitMultiplier: 1.5,
```

Comment:
Should we use `DefaultGasLimitMultiplier` here to have a single source of truth?